runtime/test-runtime/src/lib.rs

```diff
@@ -433,7 +433,7 @@ impl claims::Config for Runtime {
 }
 
 parameter_types! {
-	pub storage MinVestedTransfer: Balance = 100 * DOLLARS;
+	pub const MinVestedTransfer: Balance = 100 * DOLLARS;
```

Comment:
why is this change being made?

sounds like instead you should be placing something within a externalities environment instead of doing this